test/adapter/json_api/linked_test.rb

```diff
@@ -153,11 +153,11 @@ def test_include_multiple_posts_and_linked
             serializer = BioSerializer.new @bio1
             adapter = ActiveModel::Serializer::Adapter::JsonApi.new(
               serializer,
-              include: ['author', 'author.posts']
+              include: [author: [:posts]]
```

Comment:
I left some of the tests with the string syntax. We could add more, although I think the right place to test this is in the `include_args_to_hash` tests.